test/functional/page_objects/visual_builder_page.ts

```diff
@@ -293,6 +301,44 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
       await el.pressKeys(browser.keys.RETURN);
       await PageObjects.header.waitUntilLoadingHasFinished();
     }
+
+    /**
+     * check that table visualization is visible and ready for interact
+     *
+     * @returns {Promise<void>}
+     * @memberof VisualBuilderPage
+     */
+    public async checkTableExisting(): Promise<void> {
+      await testSubjects.existOrFail('visualizationLoader');
+      await testSubjects.existOrFail('noTSVBDataMessage');
+    }
```

Comment:
Several thoughts here:
1. `checkTableTabIsPresent` is better to follow the naming pattern for other tabs.
2. It will fail if the table has the data on opening the tab and it can be the case, right? I suggest looking for a reliable way that will work for both table rendered/not shown to check tab is opened.